Classes/Domain/Repository/EventRepository.php

```diff
@@ -113,14 +113,17 @@ public function findWibaByContact($contact, $category = 0, $bExcludeCategory)
      *
      * @return array The found Event Objects
      */
-    public function findEventByContact($contact)
+    public function findEventByContact($contact, $category = null)
     {
         $query = $this->createQuery();
 
         $constraints = [];
         $constraints[] = $query->equals('contact', $contact);
         $constraints[] = $query->equals('genius_bar', 0);
         $constraints[] = $query->equals('cancelled', 0);
+        if ($category != null) {
+            $constraints[] = $query->in('categories.uid', explode(',', $category));
```

Comment:
No, cause the categories from flexform will be comma separated string, if there is more than one selected.